vnc_assume_role_policy_doc.tf

```diff
@@ -0,0 +1,32 @@
+# ------------------------------------------------------------------------------
+# Create an IAM policy document that allows the EC2 instances in the
```

Comment:
@jsf9k Does this wording sound better to you?
```
# Create an IAM policy document that allows EC2 instances and any user in the
# assessment account to assume the role this policy is attached to.
```

Reply:
Why is it a user in the assessment account and not the users account?

Reply:
Arg, you are right- there are no users in the assessment account.  

I don't think we even need users to be able to assume this role- it's really just so the cloud-init script on the Guacamole instance can talk to the SSM Parameter Store.  So it will just become:
```
# Create an IAM policy document that allows EC2 instances 
# to assume the role this policy is attached to.
```
And I will remove the second statement in the policy entirely.

Reply:
Cleaned up in https://github.com/cisagov/cool-assessment-terraform/pull/5/commits/f022950aadf7c6b11074df026198c284cc59fe49.

Reply:
@jsf9k - After I deployed this PR, I noticed that one of the cloud-init scripts was failing with this error:
```
botocore.exceptions.ClientError: An error occurred (AccessDenied)
when calling the AssumeRole operation:
User: arn:aws:sts::051445540311:assumed-role/guacamole_instance_role_default/i-059a2b3e51732a452
is not authorized to perform: sts:AssumeRole on resource:
arn:aws:iam::207871073513:role/ParameterStoreReadOnly-env0-VNC
```
It seems that the statement I removed in https://github.com/cisagov/cool-assessment-terraform/commit/f022950aadf7c6b11074df026198c284cc59fe49 was allowing the Guacamole instance to successfully assume the `ParameterStoreReadOnly-env0-VNC` role.  I'm pretty sure that statement was allowing any user from the `env0` account to assume the role and that apparently includes the Guacamole instance profile.

I have tested an updated policy that only allows the Guacamole instance profile role to assume the `ParameterStoreReadOnly-env0-VNC` role, so I will be creating a PR with that change soon.